.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
I've added individual script lines to perform an install and then perform the unit tests before generating the javadoc items. Is this ok?